### PR TITLE
fix(Text Column Formatting): Formatting not working in Measures in Rows VBI-5575

### PR DESCRIPTION
**Pull request by vishnu-varthanan**
# Changes Made On Draft Editor

1.  [x] Editor
2.  [ ] Toolbar

# Purpose of change

-   [ ] NEW FEATURE IMPLEMENTATION
-   [x] CODE QUALITY IMPROVEMENT
-   [x] BUG FIX
-   [ ] Version Updation

# Description of change

# Reviewer

### CODE REVIEW

**Review by hb1998 — CHANGES_REQUESTED** [review]
after the required changes are made, make sure you get a new version from devaraj and update it in ibcslibrary, and test the fix once

File: src/DraftApi.ts [hunk 1]
```diff
@@ -10,7 +10,17 @@ export default class DraftApi {
...
         let editorState = EditorState.createWithContent(convertFromHTMLString(htmlString));
         editorState = selectAll(editorState);
-        editorState = RichUtils.toggleInlineStyle(editorState, formatType.toUpperCase());
+        const currentSelection = editorState.getSelection();
+        const contentState = editorState.getCurrentContent();
+        const startkey = currentSelection.getStartKey();
+        const startOffset = currentSelection.getStartOffset();
+        const blockWithSelection = contentState.getBlockForKey(startkey);
+        const inlineStyles = blockWithSelection.getInlineStyleAt(startOffset);
+        const applyFormat = formatValue && !inlineStyles.has(formatType.toUpperCase());
```
[the remark this answers is not in the document]
Comment (vishnu-varthanan) [reply src/DraftApi.ts]: Okay

**Comment by nsdevaraj** [comment]
Send PR to 2.4

**Review by hb1998 — APPROVED** [review]
LGTM

**Pull request merged**